packages/client/lib/sync/fetcher/fetcher.ts

```diff
@@ -58,12 +74,17 @@ export class Fetcher extends Readable {
     super({ ...options, objectMode: true })
 
     this.config = options.config
+    this.chain = options.chain
     this.pool = options.pool
+
+    this.first = options.first
+    this.count = options.count
+
     this.timeout = options.timeout ?? 8000
     this.interval = options.interval ?? 1000
     this.banTime = options.banTime ?? 60000
     this.maxQueue = options.maxQueue ?? 16
-    this.maxPerRequest = options.maxPerRequest ?? 128
+    this.maxPerRequest = options.maxPerRequest ?? 25
```

Comment:
Any reason why we change this to 25?

Reply:
Yes, since VM execution makes everything substantially slower I reduced this to have chunks which terminate in a reasonable timeframe (something under 1 minute)